src/components/aboutUs/AboutUs.js

```diff
@@ -0,0 +1,70 @@
+import React, { useRef, useState, useEffect } from "react"
+import { Form, Button, Card } from "react-bootstrap"
+import { useHistory } from "react-router-dom"
+import "react-datepicker/dist/react-datepicker.css";
+import { useAuth } from "../../contexts/AuthContext"
+import Typography from "@material-ui/core/Typography";
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+
+import {
+    Alert,
+    AlertIcon,
+    AlertTitle,
+    AlertDescription,
+    Box,
+    Input,
+    FormControl,
+    FormLabel,
+    FormErrorMessage,
+    FormHelperText,
+    Select,
+    Text,
+    Image,
+    Link
+} from '@chakra-ui/react';
+
+import NavigationBar from '../common/NavigationBar';
+
+import GOOSE_IMAGE from "../../images/Goose.png"
+
+import "../../styles/form-style.css";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const CreateResource = () => {
+
+    return (
+        <div>
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 100, left: '50%', transform: 'translateX(-50%)' }}>
+                <Card style={{ padding: '16px' }}>
+                    <Text noOfLines={1} align="center" fontSize='4xl'>We are HonkedIn.</Text>
+                    <Image
+                        src={GOOSE_IMAGE}
+                        mx="auto"
+                        boxSize="200px"
+                        alt="Goose Image"
+                    />
+                    <Text align="center" fontSize='2xl' as='i'>Made by students, for students.</Text>
+                    <Text align="center" fontSize='2xl'>We aim to connect like-minded scholars at University of Waterloo.</Text>
+                </Card>
+                <br></br>
+                <Card style={{ padding: '16px' }}>
+                    <Text align="center" fontSize='2xl'>Want to learn more about how HonkedIn was built?</Text>
+                    <Link href='https://github.com/Safewaan/HonkedIn' isExternal align="center" target="_blank">
+                        Find out more here!<ExternalLinkIcon mx='2px' />
+                    </Link>
+                    <br></br>
+                    <Text align="center" fontSize='2xl'>Want to see HonkedIn in action?</Text>
+                    <Link href='https://youtu.be/01CqZoxI3ps' isExternal align="center" target="_blank">
+                        Check this out!<ExternalLinkIcon mx='2px' />
```

Comment:
these links aren't centered:
![image](https://user-images.githubusercontent.com/57375371/226491033-db908fd0-9de2-4f19-8347-9e2cd46098d5.png)

Reply:
oh for me they are
<img width="775" alt="Screenshot 2023-03-20 at 9 16 13 PM" src="https://user-images.githubusercontent.com/76895894/226497841-3c11db08-9ada-469e-970b-b9c9068f699a.png">

Reply:
might be a cache issue, I'll try relaunching the app when I test

Reply:
kk lmk